### PR TITLE
Move CMSSW_7_4_X to use CMSSW_7_4_ROOT6_X.

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by ktf** [comment]
Trying this in DEVEL_X before we merge into baseline.

File: DataFormats/JetReco/src/classes_def_4.xml [hunk 1]
```diff
@@ -137,4 +137,23 @@
    <class name="edm::Association<reco::FFTCaloJetCollection>" />
    <class name="edm::Wrapper<edm::Association<reco::FFTCaloJetCollection> >" />
 
+
+
```
Comment (slava77) [inline DataFormats/JetReco/src/classes_def_4.xml]: CATopJetProperties was moved to DataFormats/BTauReco
why is it coming up here? Need to remove

Comment (ktf) [inline DataFormats/JetReco/src/classes_def_4.xml]: Fixed in #8432 8432

**Comment by slava77** [comment]
I didn't notice anything else peculiar

**Pull request merged**